src/main/java/seedu/address/storage/JsonAdaptedPerson.java

```diff
@@ -92,6 +92,11 @@ public Person toModelType() throws IllegalValueException {
         if (!Email.isValidEmail(email)) {
             throw new IllegalValueException(Email.MESSAGE_CONSTRAINTS);
         }
+
+        if (sid < 0) {
```

Comment:
Is id 0 valid?

Reply:
No it is 1, 2, 3 etc. 0 is not valid.

Reply:
I pushed the new change already. Cheers!